### ISSUE TITLE
Training  difference between 2 work

### ISSUE THREAD

**Issue by 009deep** [clovaai/voxceleb_trainer#89]
@joonson In [paper1](https://arxiv.org/pdf/2010.15809.pdf) you have the minimum EER of 0.88 vs 1.18 in [paper2](https://arxiv.org/pdf/2009.14153.pdf). 

What is main  difference in  training there?  It looks same per table in  term  of architecture and loss. Also, I think weight on github is from paper2. Could you share weight from paper1?

**Comment by joonson** [comment]
It was made possible by improvements from a much larger batch size from FP16 training. I also think that (without evidence) the 0.88 one had a 'lucky' random seed.

**Comment by 009deep** [comment]
Ah,  I see. So is that separate codebase with  use of autocast?  Could you provide  sample of how to properly use autocast with training here?

**Comment by joonson** [comment]
It's merged into master now. You need to use the `--mixedprec` flag.

**Comment by 009deep** [comment]
Ah, I see. I haven't used that merge yet, I'll try out. Could you provide new weights and setting to verify result?

**Comment by joonson** [comment]
[Here](https://www.robots.ox.ac.uk/~joon/data/baseline_v2_smproto.model) is the model that produces 0.88% EER on Vox1 cleaned. 

@009deep  Also pls note that `2010.15809` tests using the cleaned version of Vox1 and `2009.14153` tests using the original version of Vox1. This will make a difference of at least 0.1% EER.

**Comment by llearner** [comment]
> [Here](https://www.robots.ox.ac.uk/~joon/data/baseline_v2_smproto.model) is the model that produces 0.88% EER on Vox1 cleaned.
> 
> @009deep Also pls note that `2010.15809` tests using the cleaned version of Vox1 and `2009.14153` tests using the original version of Vox1. This will make a difference of at least 0.1% EER.

thanks very much!

**Comment by 009deep** [comment]
Thank you @joonson .

While using **_ResNetSE34V2_** , I get following warning messages:
__S__.bn_last.weight is not in the model.
__S__.bn_last.bias is not in the model.
__S__.bn_last.running_mean is not in the model.
__S__.bn_last.running_var is not in the model.
__S__.bn_last.num_batches_tracked is not in the model.

Is there **_bn_last_**  layer which is aded  in training for this model and not part of master here?

**Comment by joonson** [comment]
There was an optional output batchnorm in the experimental code, but it has been deleted since. You can ignore this warning.

**Comment by 009deep** [comment]
Thank you so much for your guidance and help. I  could verify result  with this weight.

One thing I'd note as I have commented previously, it improves result for voxceleb test set but  worsens for in-house (out-of-domain, non youtube) dataset compared with `2009.14153`.  I'll raise separate discussion for ideal way for using non-domain data such  librispeech with this training. Thanks.
